example/rails-51/config/initializers/bugsnag.rb

```diff
@@ -1,4 +1,5 @@
 Bugsnag.configure do |config|
-  config.api_key = "YOUR_API_KEY"
-  config.auto_capture_sessions = true
+  config.api_key = "12312312312312312312312312312312"
```

Comment:
Unless there's a good reason, this should remain as "YOUR_API_KEY" for additional clarity

Reply:
Yeah, that's my bad.